Icon-Legend.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
```

Comment:
I would give it a specific title

Reply:
I dont play on adding this html, Just the specific code to the main pages.

Reply:
> 
> 
> I would give it a specific title

I just did a sample page as title, but I just plan to import the data instead of using this HTML